test/commands_map.ts

```diff
@@ -356,6 +356,14 @@ var commands_map: CommandValuesMap = {
             sendActivity(conversationId, server_content.receipt_card);
         }
     },
+    "text-empty": {
+        client: function () {
+            return true;
```

Comment:
can you verify that there is a bubble? Test should fail if we run against current code.